tvshows/src/components/shared/ShowsList/ShowsList.tsx

```diff
@@ -25,11 +21,11 @@ export const ShowsList = ({ route, getter }: IWhichList) => {
     return <div>Oops, something went wrong...</div>;
   }
 
-  const { shows } = showsListResponse;
+  const { shows } = showsListResponse as { shows: IShow[] };
```

Comment:
You can add this typing to the useSwr call with a generic instead of casting here.